src/openforms/conf/base.py

```diff
@@ -47,6 +47,14 @@
     ("nl", _("Dutch")),
 ]
 
+LANGUAGE_COOKIE_HTTPONLY = True
+
+LANGUAGE_COOKIE_NAME = "openforms_language"
```

Comment:
Not sure if we want this to be configurable

Reply:
nope, fixed value is fine (session cookie name is also fixed)